Web/starter_project_vanguard/styles/globals.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
```

Comment:
is this the best way to import the fonts ?

I have seen some people define them i think in the tailwind configs. might be worth looking into